### PR TITLE
DB Migration - Permission Management Enhancements

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/resources/config/liquibase/changelog/20200521100000_added_user_group_permission_data.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet author="unknown" id="insert-example-2">
```
Comment (admin-vizcentric) [inline src/main/resources/config/liquibase/changelog/20200521100000_added_user_group_permission_data.xml:4]: Please fill out this attributes correctly and meaningful.

Comment (khushbu-maurya) [reply src/main/resources/config/liquibase/changelog/20200521100000_added_user_group_permission_data.xml]: done

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Pull request merged**